package.json

```diff
@@ -34,6 +37,10 @@
 		"webpack-cli": "^2.0.10",
 		"xo": "*"
 	},
+	"prettier": {
+		"bracketSpacing": false,
+		"singleQuote": true
+	},
```

Comment:
This doesn't belong here. The project doesn't use Prettier. You should rather just locally disable Prettier for this project.

Reply:
Disabling Prettier in my editor would be very counterproductive for me. This was just to keep the formatting consistent with XO and it doesn't make a difference to anything else.